<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="0" total="5" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="My Sample Suite" duration-ms="63463" started-at="2019-10-09T01:37:33Z" finished-at="2019-10-09T01:38:37Z">
    <groups>
    </groups>
    <test name="Login Test" duration-ms="29052" started-at="2019-10-09T01:37:33Z" finished-at="2019-10-09T01:38:02Z">
      <class name="testcases.LoginTest">
        <test-method status="PASS" signature="init()[pri:0, instance:testcases.LoginTest@42e99e4a]" name="init" is-config="true" duration-ms="462" started-at="2019-10-09T01:37:33Z" finished-at="2019-10-09T01:37:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="doLogin(java.util.Hashtable)[pri:0, instance:testcases.LoginTest@42e99e4a]" name="doLogin" duration-ms="18062" started-at="2019-10-09T01:37:34Z" data-provider="getData" finished-at="2019-10-09T01:37:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=ashishthakur1983, ExpectedResult=Login_Failure, Password=pass@1234, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Got actual result as Login_Success]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Got actual result as Login_Success
at org.testng.Assert.fail(Assert.java:96)
at com.qtpselenium.keywords.GenericKeywords.reportFailure(GenericKeywords.java:111)
at com.qtpselenium.keywords.AppKeywords.validateLogin(AppKeywords.java:39)
at com.qtpselenium.driver.DriverScript.executeKeywords(DriverScript.java:52)
at testcases.LoginTest.doLogin(LoginTest.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="quit()[pri:0, instance:testcases.LoginTest@42e99e4a]" name="quit" is-config="true" duration-ms="10170" started-at="2019-10-09T01:37:52Z" finished-at="2019-10-09T01:38:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="SKIP" signature="doLogin(java.util.Hashtable)[pri:0, instance:testcases.LoginTest@42e99e4a]" name="doLogin" duration-ms="4" started-at="2019-10-09T01:38:02Z" data-provider="getData" finished-at="2019-10-09T01:38:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=ashishthakur1983, ExpectedResult=Login_Success, Password=pass@1234, Runmode=N, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test as Runmmode is N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test as Runmmode is N
at testcases.LoginTest.doLogin(LoginTest.java:45)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="init()[pri:0, instance:testcases.LoginTest@42e99e4a]" name="init" is-config="true" duration-ms="0" started-at="2019-10-09T01:38:02Z" finished-at="2019-10-09T01:38:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="quit()[pri:0, instance:testcases.LoginTest@42e99e4a]" name="quit" is-config="true" duration-ms="123" started-at="2019-10-09T01:38:02Z" finished-at="2019-10-09T01:38:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="init()[pri:0, instance:testcases.LoginTest@42e99e4a]" name="init" is-config="true" duration-ms="0" started-at="2019-10-09T01:38:02Z" finished-at="2019-10-09T01:38:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="doLogin(java.util.Hashtable)[pri:0, instance:testcases.LoginTest@42e99e4a]" name="doLogin" duration-ms="3" started-at="2019-10-09T01:38:02Z" data-provider="getData" finished-at="2019-10-09T01:38:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=U1, ExpectedResult=Login_Failure, Password=P1, Runmode=N, Browser=Edge}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test as Runmmode is N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test as Runmmode is N
at testcases.LoginTest.doLogin(LoginTest.java:45)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="quit()[pri:0, instance:testcases.LoginTest@42e99e4a]" name="quit" is-config="true" duration-ms="119" started-at="2019-10-09T01:38:02Z" finished-at="2019-10-09T01:38:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- Login Test -->
    <test name="Create Portfolio Test" duration-ms="15867" started-at="2019-10-09T01:38:03Z" finished-at="2019-10-09T01:38:18Z">
      <class name="testcases.CreatePortfolioTest">
        <test-method status="PASS" signature="doLogin(java.util.Hashtable)[pri:0, instance:testcases.CreatePortfolioTest@436813f3]" name="doLogin" duration-ms="9914" started-at="2019-10-09T01:38:03Z" data-provider="getData" finished-at="2019-10-09T01:38:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{PortfolioName=Temp2019, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="init()[pri:0, instance:testcases.CreatePortfolioTest@436813f3]" name="init" is-config="true" duration-ms="0" started-at="2019-10-09T01:38:03Z" finished-at="2019-10-09T01:38:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="quit()[pri:0, instance:testcases.CreatePortfolioTest@436813f3]" name="quit" is-config="true" duration-ms="5946" started-at="2019-10-09T01:38:12Z" finished-at="2019-10-09T01:38:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- testcases.CreatePortfolioTest -->
    </test> <!-- Create Portfolio Test -->
    <test name="Add Stock Test" duration-ms="18228" started-at="2019-10-09T01:38:18Z" finished-at="2019-10-09T01:38:37Z">
      <class name="testcases.AddStockTest">
        <test-method status="PASS" signature="init()[pri:0, instance:testcases.AddStockTest@49e53c76]" name="init" is-config="true" duration-ms="1" started-at="2019-10-09T01:38:18Z" finished-at="2019-10-09T01:38:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="doLogin(java.util.Hashtable)[pri:0, instance:testcases.AddStockTest@49e53c76]" name="doLogin" duration-ms="18097" started-at="2019-10-09T01:38:18Z" data-provider="getData" finished-at="2019-10-09T01:38:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{PurchasePrice=500.0, Runmode=Y, Quantity=100.0, PurchaseDate=12-04-2019, StockName=Tata Steel Ltd, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="quit()[pri:0, instance:testcases.AddStockTest@49e53c76]" name="quit" is-config="true" duration-ms="122" started-at="2019-10-09T01:38:37Z" finished-at="2019-10-09T01:38:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- testcases.AddStockTest -->
    </test> <!-- Add Stock Test -->
  </suite> <!-- My Sample Suite -->
</testng-results>
